assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js

```diff
@@ -105,6 +112,28 @@ const DashboardIdeasWidget = ( {
 		[ setHash, setActiveTabIndex ]
 	);
 
+	// Any time the pagination value changes, scroll to the top of the container.
+	useUpdateEffect( () => {
+		const tabBarRectangle = tabBarHeaderRef.current.getBoundingClientRect();
+
+		const isOnScreen =
+			tabBarRectangle.top >= 0 &&
+			tabBarRectangle.left >= 0 &&
+			tabBarRectangle.bottom <=
+				( global?.innerHeight ||
+					global?.document?.documentElement?.clientHeight ) &&
+			tabBarRectangle.right <=
+				( global?.innerWidth ||
+					global?.document?.documentElement?.clientWidth );
+
+		if ( ! isOnScreen ) {
+			global.window.scrollTo( {
+				top: getIdeaHubContainerOffset( tabBarRectangle.top ),
```

Comment:
This function results in an improper scroll position on mobile due to what seems to be the added offset of the wp-admin bar height but this shouldn't be added in this case because it isn't offsetting anything. Only the sticky SK header needs to be accounted for.

Position after scrolling: 
![image](https://user-images.githubusercontent.com/1621608/130781400-331564ca-52e2-4145-ab06-c7c23f46ac94.png)

Reply:
I've removed the WP Admin from the calculation and it works much better, thanks 👍🏻